### DISCUSSION TITLE
Matching multiple paths with the ondemand parser

### DISCUSSION THREAD

**Discussion by amunra** [simdjson/simdjson#2192] [General]
I would like to extract multiple paths from a JSON document efficiently.
For example, from a single `pets.json` document, I might want to extract `.pets[0].owner.name` and `.pets[0].owner.age`.
What's the best way to achieve this with the current API? I am trying to avoid starting from the start of the document every time.

**Comment by lemire** [comment]
I’d recommend using our On Demand API directly.
